cypress/integration/smoke-test.js

```diff
@@ -1,8 +1,13 @@
 beforeEach(function () {
     cy.fixture('postcodes.json').as('postcodes')
-    Cypress.Cookies.defaults({
-        whitelist: ['_get_help_to_retrain_session', 'seen_cookie_message']
-    });
+    // Cypress.Cookies.defaults({
```

Comment:
Why is this commented out? I think we still need the seen cookie message?

Reply:
This was commented out as whitelisting these cookies for some reason didn't allow for me to clear them in case it failed and needed to be re-run. I left it commented out in case we needed to come back to them.